rewrite-gradle/src/main/resources/META-INF/rewrite/gradle-wrapper/versions.csv

```diff
@@ -0,0 +1,183 @@
+version,gradlew,gradlewBat
```

Comment:
question: Why a CSV with Adler32 versus a directory structure?

Ex:
```
/META-INF/rewrite/gradle-wrapper/
    4.10.3/
        gradlew
        gradlew.bat
    5.6.4/
        gradlew
        gradlew.bat
    ...
```

I'm guessing some of this is that having the CSV makes finding the versions a little bit easier.

Reply:
Because there is so many versions for which the wrapper script didn't change. So the proposed structure keeps only one copy of any given iteration of the wrapper script.